src/hammer-vlsi/synthesis/genus/__init__.py

```diff
@@ -3,21 +3,36 @@
 #
 #  hammer-vlsi plugin for Cadence Genus.
 #
-#  Copyright 2017 Edward Wang <edward.c.wang@compdigitec.com>
+#  Copyright 2017-2018 Edward Wang <edward.c.wang@compdigitec.com>
 
-from hammer_vlsi import HammerTool
+from hammer_vlsi import HammerToolStep
 from hammer_vlsi import CadenceTool
 from hammer_vlsi import HammerSynthesisTool
 from hammer_vlsi import HammerVLSILogging
 
-import hammer_tech
-
-from functools import reduce
-from typing import Callable, Dict, List, Iterable, Any
+from typing import Dict, List, Any
 
 import os
 
+
 class Genus(HammerSynthesisTool, CadenceTool):
+    def fill_outputs(self) -> bool:
```

Comment:
Not sure I understand the name of this function. It seems to just check_outputs, not create or fill them.